install_config/aggregate_logging.adoc

```diff
@@ -724,6 +724,10 @@ named to match the `openshift_logging_es_pvc_prefix` setting, which defaults to
 exists already, it is used; if not, and `openshift_logging_es_pvc_size` has been
 specified, it is created with a request for that size.
 
+The {product-title} installer creates the persistent volumes, based on the `openshift_logging_storage_*` parameters 
+and the xref:#deploying-the-efk-stack[EFK deployment] creates PVCs, using the openshift_logging_es_pvc_ paramters.
```

Comment:
instead of `[EFK deployment]` can we use something like `[{product-title} Logging installer]`?